docs/project_insights

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Great, thanx for this!
A couple of amendments:
docs files have a `.md` extension.
What I find the most time-consuming is navigating the files, finding the respective modules, what templates or react components are related to, and keep an overview of the relevant modules in my head. So here would be good to add a section of `Structure` and give the paths to the files that contain the relevant code. 
E.x: 
```
adhocracy4 
adhocracy4/projects/mixins.py
DisplayProjectOrModuleMixin() -> get_context_data(), 

adhocracy-plus
apps/projects/insights.py -> functions create_insights(), create_insight()
apps/projects/models.py -> model ProjectInsight, static method ProjectInsight.update_context()
[etc...]
```
also the templates, they are two of these iirc. can you lookup the merged PR again for this? would be very helpful.

Reply:
@m4ra I mark this as resolved for now. The "new documentation" could be part of the documentation story?